deploy/kubernetes/overlays/prow-head-template/node_images.yaml

```diff
@@ -9,4 +9,8 @@ spec:
         - name: gce-pd-driver
           image: PROW_GCEPD_IMAGE:latest
         - name: csi-driver-registrar
-          image: gcr.io/gke-release/csi-driver-registrar:v0.4.1-gke.0
\ No newline at end of file
+<<<<<<< HEAD
```

Comment:
?

Reply:
very weird, my editor wasn't showing that as a thing. but when i went in with vim I could see the merge conflict. Fixed